### PR TITLE
Marketplace: Add landingPage sections and tags

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33809
fixes: https://github.com/demisto/etc/issues/34238

## Description
Added landingPage sections

### Test upload build:
https://app.circleci.com/pipelines/github/demisto/content/72382/workflows/4b0db849-d836-4271-8c34-ebe62e9038d1

### Index with changes:

https://console.cloud.google.com/storage/browser/marketplace-dist-dev/upload-flow/builds/marketplace-sections/298128/content/packs;tab=objects?project=oproxy-dev&prefix=index&forceOnObjectsSortingFiltering=false

HelloWorld was tested with the `Featured` section. both `searchRank` and `tags` entries were updated successfully.

### CODE REVIEW

File: Tests/Marketplace/Tests/marketplace_services_test.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def test_validate_all_fields_of_parsed_metadata(self, dummy_pack, dummy_pack_met
         assert parsed_metadata['currentVersion'] == '2.3.0'
         assert parsed_metadata['versionInfo'] == "dummy_build_number"
         assert parsed_metadata['commit'] == "dummy_commit"
-        assert parsed_metadata['tags'] == ["tag number one", "Tag number two", "Use Case"]
+        assert set(parsed_metadata['tags']) == {"tag number one", "Tag number two", "Use Case"}
```
Comment (yaakovi) [inline Tests/Marketplace/Tests/marketplace_services_test.py:91]: add validation for the list length too, in the new form there could be duplicate tags

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -601,9 +602,29 @@ def _get_search_rank(tags, certification, content_items):
 
         return search_rank
 
+    def _handle_pack_tags(self, landing_page_sections: dict, user_tags: list = None) -> list:
+        """
+        Build the pack's tag list according to the user metadata and the landingPage sections file.
+        Args:
+            user_tags: user metadata that was created in pack initialization.
```
Comment (yaakovi) [inline Tests/Marketplace/marketplace_services.py:609]: add `landing_page_sections` too

File: Tests/Marketplace/marketplace_services.py [hunk 3]
```diff
@@ -601,9 +602,29 @@ def _get_search_rank(tags, certification, content_items):
 
         return search_rank
 
+    def _handle_pack_tags(self, landing_page_sections: dict, user_tags: list = None) -> list:
```
Comment (yaakovi) [inline Tests/Marketplace/marketplace_services.py:605]: it can be a static function

**Review by yaakovi — APPROVED** [review]
nice work

File: Tests/Marketplace/Tests/marketplace_services_test.py (hunk 1 above)
Comment (Bargenish) [reply Tests/Marketplace/Tests/marketplace_services_test.py:91]: I used set to build the tags list so no duplicates are expected.

**Pull request merged**